docs/pages/index.mdx

```diff
@@ -57,9 +63,223 @@ endpoints, infrastructure assets, and AI agents.
+        }
+      ]
+    },
+    {
+      id: 'identity-governance',
+      title: 'Identity Governance',
+      href: './identity-governance/',
+      description: 'Manage identities by enforcing principles of least privilege and zero trust',
+      iconColor: '#EEEAFA',
+      iconComponent: IGSvg,
+      features: [
+        {
+          title: 'Request Temporary Elevated Access',
+          description: 'Eliminate standing privileges w/ just-in-time access',
+          href: './identity-governance/access-requests/'
+        },
+        {
+          title: 'Manage Standing Access for Teams',
+          description: 'Sync IdP groups to roles w/ automated reviews',
+          href: './identity-governance/okta/app-and-group-sync/'
+        },
+        {
+          title: 'Require Managed Devices for Access',
+          description: 'Guarantee access only from trusted devices',
+          href: './identity-governance/device-trust/'
+        },
+        {
+          title: 'Instantly Lock Identities & Sessions',
+          description: 'Lock compromised users and resources',
+          href: './identity-governance/locking/'
+        },
+        {
+          title: 'Integrate w/your Identity Provider(s)',
+          description: 'Okta, Entra ID, and Sailpoint w/SCIM group sync',
+          href: './identity-governance/okta/'
+        },
+        {
+          title: 'Federate Users to External Services',
+          description: 'Use Teleport as SAML IdP to 3rd-party apps',
+          href: './admin-guides/access-controls/idps/'
+        },
+        {
+          title: 'Monitor for Risky Access',
+          description: 'Identify risky access patterns and behaviors',
+          href: './identity-governance/access-monitoring/'
+        }
+      ]
+    },
+    {
+      id: 'identity-security',
+      title: 'Identity Security',
+      href: './identity-security/',
+      description: 'Visualize access paths and identify security risks across your infrastructure',
+      iconColor: '#EEEAFA',
+      iconComponent: ISSvg,
+      features: [
+        {
+          title: 'Expose Hidden Access Risks',
+          description: 'Scan for SSH keys, repo access, and more',
+          href: './identity-security/integrations/ssh-keys-scan'
+        },
+        {
+          title: 'Identify Over-Privileged Users',
+          description: 'Identify users with excessive standing privileges',
+          href: './identity-security/policy-how-to-use/#dashboard-standing-privileges'
+        },
+        {
+          title: 'Monitor Changes to Critical Resources',
+          description: 'Monitor, alert on, and visualize access changes',
+          href: './identity-security/crown-jewels'
+        },
+        {
+          title: 'Alert on Anomalous Activity',
+          description: 'AI alerting on risky behaviors in your infrastructure',
+          href: './identity-governance/access-monitoring'
+        },
+        {
+          title: 'Unify Logs from Key Systems',
```

Comment:
This makes us sound more like a Splunk competitor. Is that what we're going for here?